inference-engine/src/transformations/src/transformations/convert_opset4_to_opset3/convert_sequences_to_sequences_ie.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Please move this transformations files to convert_opset1_to_legacy

Reply:
done